tests/functional/object/rgw/test_obc_quota.py

```
@@ -121,6 +121,96 @@ def test_rgw_obc_quota(
+            copy_random_individual_objects(
+                awscli_pod_session,
+                pattern="object-",
+                file_dir=test_dir,
+                target=full_bucket_path,
+                amount=upload_amount,
+                s3_obj=obc_obj,
+                ignore_error=False,
+            )
+        except CommandFailed as e:
+            if err_msg in e.args[0]:
+                logger.info(f"Size quota {quota} worked as expected!!")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)